app/mailers/order_mailer.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class OrderMailer < ApplicationMailer
+  def confirmation_send(user, order)
+    @order = order
+    @user = user
+    mail(to: @user.email,
+         subject: 'Confirm your purchase',
+         template_path: 'orders/mailers',
+         template_name: 'confirm')
+  end
+
+  def treating_send(user, order)
+    @order = order
+    @user = user
```

Comment:
@order = order
 @user = user

this is used in many methods, can we move it somewhere?